package.json

```diff
@@ -20,6 +20,7 @@
   ],
   "dependencies": {
     "express": "^4.8.5",
+    "find-port": "^1.0.1",
```

Comment:
it's the new package.json license format. No more urls in licenses. you get a warning on install without this change.